File: Calcular_Salario/calcular_salario.cs
using System;

public class Desafio
{
    public static void Main()
    {
        //Lê os valodres de Endrada:
        float valorSalario = float.Parse(Console.ReadLine());
        float valorBeneficios = float.Parse(Console.ReadLine());

        float valorImposto = 0;
        if (valrSalario >= 0 && valorSalario <= 1100)
        {
            //Atribui a aliquota de 5% mediante o salário:
            valorImposto = 0.05f * valorSalario;
        }
        else if (valorSalario >= 1100.01 && valorSalario <= 2500.00)
        {
            valorImposto = 0.10F * valorSalario;
        }
        else (valorSalario >= 2500.00)
            {
                valorImposto = 0.15F * valorSalario;
        }
        //To-do Criar as demais condições para as aliquotas de 10.00% e 15.00%.

        //Calcula e imprime a Saída (com 2 casas decimais):
        float saida = valorSalario - valorImposto + valorBeneficios;
        Console.WriteLine(saida.ToString("0.00"));
    }
}
    

File: Calcular_Salario/calcular_salario.js
//Lê os valores de Endrada:
const valorSalario = parseFloat(gets());
const valorBeneficios = parseFloat(gets());

//Calcula o imposto através da função "calcularImpoto":
const valorImposto = calcularImposto(valorSalario);
//Calcula e imprime a Saída (com 2 casas decimais):
const saida = valorSalario - valorImposto + valorBeneficios;
print(saida.toFixed(2));

//Função útil para o calculo do imposto (baseado nas alíquotas).
function calcularImposto(salario) {
    var aliquota;
    if (salario >= 0 && salario <= 1100) {
        aliquota = 0.05;
    } else if (salario >= 1100 && salario <= 2500) {
        aliquota = 0.10;
    } else (salario > 2500)
        aliquota = 0.15;
        
    //TO-DO Criar as demais condições para as aliquotas de 10.00% e 15.00%.
    return aliquota * salario;
}
    